index.js

```diff
@@ -1,9 +1,9 @@
-module.exports = function reverse (a) {
-  var length = a.length
-  var buffer = new Buffer(length)
+module.exports = function reverse (src) {
+  var buffer = new Buffer(src.length)
 
-  for (var i = 0, j = length - 1; i < length; ++i, --j) {
-    buffer[i] = a[j]
+  for (var i = 0, j = src.length - 1; i <= j; ++i, --j) {
+    buffer[i] = src[j]
+    buffer[j] = src[i]
```

Comment:
@dominictarr inplace version located in file [inplace.js](https://github.com/crypto-browserify/buffer-reverse/blob/a4124fb8f46f94bbca4e41acc2124e0ac8bddb44/inplace.js)
@dcousens maybe make sense use inplace by default?